vhdbuilder/packer/cleanup.sh

```diff
@@ -21,13 +23,15 @@ if [[ -n "$PKR_RG_NAME" ]]; then
     echo "Packer resource group already successfully deleted"
   fi
 fi
+capture_benchmark "delete_packers_rg"
```

Comment:
could we look at augmenting the nomenclature to maybe keep the filename in the banchmaarks steps ? 

e.g. `cleanup_sh-delete_packer_rg`

Reply:
Made the change, running pipeline now